### PR TITLE
[SLICING] PBI-24: Edit Lesson Material

### PR DESCRIPTION
**Pull request by darren-ngoh**

### CODE REVIEW

File: src/resources/app/pages/course/EditCourseMaterial.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    <v-form ref="form" v-model="isValid">
+      <v-autocomplete
+        id="drop-down"
+        v-model="form.topic_id"
+        :items="items"
+        label="Topic"
+        outlined
+        :item-text="getItemText"
+        :item-value="getItemValue"
+        required
```
Comment (FXDROS) [inline src/resources/app/pages/course/EditCourseMaterial.vue:15]: nit, please change these to:
```
        id="drop-down"
        v-model="form.topic_id"
        label="Topic"
        outlined
        required
        :items="items"
        :item-text="getItemText"
        :item-value="getItemValue"
```

put all attributes that use dynamic elements together

File: src/resources/app/pages/course/EditCourseMaterial.vue [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      ></v-autocomplete>
+      <v-text-field
+        id="title-form"
+        label="Title"
+        :rules="fieldRules.concat(lengthRules)"
+        :counter="255"
+        outlined
+        v-model="form.title"
```
Comment (FXDROS) [inline src/resources/app/pages/course/EditCourseMaterial.vue:23]: same

File: src/resources/app/pages/course/EditCourseMaterial.vue [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        :item-text="getItemText"
+        :item-value="getItemValue"
+        required
+      ></v-autocomplete>
+      <v-text-field
+        id="title-form"
+        label="Title"
+        :rules="fieldRules.concat(lengthRules)"
+        :counter="255"
+        outlined
```
Comment (fxdros) [inline src/resources/app/pages/course/EditCourseMaterial.vue]: same

File: src/resources/app/pages/course/EditCourseMaterial.vue (hunk 1 above)
Comment (fxdros) [inline src/resources/app/pages/course/EditCourseMaterial.vue]: nit, please change these to:
```
        id="drop-down"
        v-model="form.topic_id"
        label="Topic"
        outlined
        required
        :items="items"
        :item-text="getItemText"
        :item-value="getItemValue"
```

put all attributes that use dynamic elements together at the end of the attributes.

**Review by Zsaschz — APPROVED** [review]
LGTM! Thanks ren now I can continue implementing edit and delete for material content

**Review by fxdros — COMMENTED** [review]
Please connect the edit page with the edit menu as well

Screen Shot 2022-04-25 at 16 15 54

**Pull request merged**